frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.tsx

```diff
@@ -98,38 +100,49 @@ export const InteractiveEmbeddingOptionCard = () => {
       label={t`PRO & ENTERPRISE`}
       description={jt`Use interactive embedding when you want to ${(
         <ExternalLink
-          href="https://www.metabase.com/blog/why-full-app-embedding"
+          href={`https://www.metabase.com/blog/why-full-app-embedding?utm_source=${plan}&utm_media=embed-settings`}
           key="why-full-app-embedding"
         >
           {t`offer multi-tenant, self-service analytics`}
         </ExternalLink>
       )} and people want to create their own questions, dashboards, models, and more, all in their own data sandbox.`}
     >
       <BoldExternalLink
-        href={
-          isEE
-            ? "https://www.metabase.com/learn/customer-facing-analytics/interactive-embedding-quick-start?utm_source=product&utm_medium=CTA&utm_campaign=embed-settings-pro-cta"
-            : "https://www.metabase.com/learn/customer-facing-analytics/interactive-embedding-quick-start?utm_source=product&utm_medium=CTA&utm_campaign=embed-settings-oss-cta"
-        }
+        href={`https://www.metabase.com/learn/customer-facing-analytics/interactive-embedding-quick-start?utm_source=${plan}&utm_media=embed-settings`}
```

Comment:
The Quickstart has since moved. I wanted to let you know ahead of time to change the URLs, but I failed. Sorry.
There is a redirect in place, so in theory there is nothing wrong, but I prefer we link to the correct URL now.

https://www.metabase.com/docs/XXX/embedding/interactive-embedding-quick-start-guide

We can keep the same UTM codes.
Hope it's not too much work, sorry.

Reply:
No worries, I opened the PR: https://github.com/metabase/metabase/pull/37523
Just a note that the util function that we have (and I used) to generate docs urls appends `.html` to the end of url.